src/main.rs

```diff
@@ -29,11 +29,8 @@ async fn main() -> Result<(), AppError> {
     setup_tracing(&config);
 
     // Set up FallbackDecoder
-    info!(
```

Comment:
Maybe `debug!`? But if not, one can always use `strace` :)

Reply:
https://github.com/blockfrost/blockfrost-platform/pull/108/files#diff-363108f50c16bc231d3b601409f5c699be655e35a405882de18815aa527a3226R39